### PR TITLE
Remove flakiness in TableMasterJournalIntegrationTest

### PR DESCRIPTION
**Pull request by yuzhu**

### CODE REVIEW

File: tests/src/test/java/alluxio/server/ft/journal/TableMasterJournalIntegrationTest.java [hunk 1]
```diff
@@ -49,9 +51,14 @@
  * Integration tests for table master functionality.
  */
 public class TableMasterJournalIntegrationTest {
+  private static final int WORKER_CAPACITY_BYTES = 200 * Constants.MB;
```
Comment (ZacBlanco) [inline tests/src/test/java/alluxio/server/ft/journal/TableMasterJournalIntegrationTest.java]: Isn't this quite a large amount?

Comment (yuzhu) [reply tests/src/test/java/alluxio/server/ft/journal/TableMasterJournalIntegrationTest.java]: transformation has a default blocksize that is quite large. This is to accommodate that.

**Comment by yuzhu** [comment]
jenkins, test this please

File: tests/src/test/java/alluxio/server/ft/journal/TableMasterJournalIntegrationTest.java (hunk 1 above)
Comment (ZacBlanco) [reply tests/src/test/java/alluxio/server/ft/journal/TableMasterJournalIntegrationTest.java]: Can we reduce the default block size? 200MB is a lot to require for a small integration test...

Comment (yuzhu) [reply tests/src/test/java/alluxio/server/ft/journal/TableMasterJournalIntegrationTest.java]: what is the negative effect of this? 
The size is determined by ParquetWriter minimum file size which is 128 MB. This is not the alluxio block size.

**Comment by yuzhu** [comment]
@ZacBlanco ping

**Comment by gpang** [comment]
alluxio-bot, merge this please

**Pull request merged**